Ruthvik-practice/Calculator.java

```diff
@@ -0,0 +1,31 @@
+import java.util.*;
+public class Calculator{
+    private static int calc(char c, int a, int b){
+        switch (c) {
+            case '+':
+                return a + b;
+            case '-':
+                return a - b;
+            case '*':
+                return a * b;
+            case '/':
+                return a / b;
```

Comment:
add check for divide by zero

Reply:
Okay sir

Reply:
Sir you did something, in my profile you repo is shwoing but not updated it is showing the repo before my commits. Can you tell me that how can I merge my folder into your repo.